### PR TITLE
Update checker info added to passed files

### PR DESCRIPTION
**Pull request by mhidas**
Now that we're deploying the compliance checker from an egg, the git commit info is no longer easily accessible. We'll just keep the release version of the core code. Also, we need to keep track of the imos plugin version.

### CODE REVIEW

**Comment by mhidas** [comment]
Note, this will fix the following errors currently seen in the pipeline logs every time a file passes the checker:
```
Jul 11 12:49:54 10-aws-syd ANMN_AM: fatal: Not a git repository (or any of the parent directories): .git
Jul 11 12:49:54 10-aws-syd ANMN_AM: fatal: Not a git repository (or any of the parent directories): .git
```

**Comment by lbesnard** [comment]
@lwgordonimos this explains the email I sent you regarding "not a git repo"

File: lib/common/netcdf-check.sh [hunk 1]
```diff
@@ -109,20 +95,17 @@ add_checker_signature() {
     local checker_version=`$NETCDF_CHECKER --version`
     local version_number=`echo $checker_version | egrep 'IOOS compliance checker version' | egrep -o '[0-9.]+$'`
 
-    local commit_hash=`_get_checker_commit_hash`
-    local last_updated=`_get_checker_commit_date`
+    local imos_version=$($NETCDF_CHECKER -l | grep ' imos ' | sed -r 's/.*\((.*)\)/\1/')
```
Comment (lbesnard) [inline lib/common/netcdf-check.sh]: could do ```pip show -V cc-plugin-imos``` ?

Comment (mhidas) [inline lib/common/netcdf-check.sh]: Yep, could do, but would still need to grep to extract just the version number from that output.

**Comment by mhidas** [comment]
This is just a quick fix for now. When we migrate everything to Python, getting at these versions will be much cleaner and easier.

File: lib/common/netcdf-check.sh (hunk 1 above)
Comment (lwgordonimos) [inline lib/common/netcdf-check.sh]: FYI, the Chef recipe tests the versions with the following one-liners... not very pretty but serviceable until able to be queried out of a Python context.

```bash
checker_ver=$(pip show compliance-checker | awk /^Version:/'{print $2}')
plugin_ver=$(pip show cc-plugin-imos | awk /^Version:/'{print $2}')
```

**Pull request merged**